h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1357,43 +1357,53 @@ def show_test_results(test_name, curr_test_val, new_test_val):
         return 0
 
 
-def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False):
+def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False, check_all=True,
```

Comment:
assert_H2OTwoDimTable_equal checks every element in table1 and table2 are equal.

However, I have a huge table1 and table2 in my new pyunit test of pyunit_wideDataset_Rotterdam_large.py.  It takes forever if we have to check every element in this case and the Jenkins test will take forever to finish.  Hence, I added an option to decide whether we want to check all table elements or a random selection of them here.